.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
```

Comment:
We bounced this as an idea for fixing an error before, is it still required?

Reply:
Yeah, it's required: see this [TravisCI](https://docs.travis-ci.com/user/docker/) documentation.